mysql_merge/insert_query_composer.py

```diff
@@ -1,11 +1,34 @@
-def compose_insert_query_template(table_name, columns):
-    columns_clause = ""
-    values_clause = ""
-    for column in columns:
-        columns_clause += "`{column_name}`, ".format(
-            column_name=column['Field'])
-        val = "{{0[{column_name}]}}, "
-        values_clause += val.format(column_name=column['Field'])
-    columns_clause = columns_clause.rstrip(", ")
-    values_clause = values_clause.rstrip(", ")
-    return "INSERT INTO `" + table_name + "` (" + columns_clause + ") VALUES(" + values_clause + ");"
+class InserQueryComposer(object):
```

Comment:
I suppose there is a typo here and below, and it should be 'InsertQueryComposer'.